### DISCUSSION TITLE
eccLevel values

### DISCUSSION THREAD

**Discussion by kaboc** [khoren93/flutter_zxing#70] [Q&A] [answered]
Documentation on `eccLevel` of WriterWidget is missing. What are valid values? I read the source code and tried the numbers 0 - 3, but the same QR code was generated with `0` / `1` / `2`. Only `3` produced a different result.

According to an external doc on the specs of QR code, the value differs depending on the version. Do I have to specify some magic number? I think it'll be much better if we can specify it using an enum, like `eccLevel: EccLevel.high`.

**Comment by khoren93** [comment, marked as answer]
Thank you for your feature request! I'm happy to announce that it is now available in the v0.10.0.
Please give it a try and let us know if you have any feedback.

**Reply by kaboc** [reply]
Wow, awesome! I'll try it together with other new features and then let you know.

**Reply by kaboc** [reply]
@khoren93 
The new enum worked as expected. Thank you very much for your hard work!

**Reply by kaboc** [reply]
The code sample in README does not seem to be up-to-date.

```dart
final img = imglib.Image.fromBytes(width, height, result.bytes);
```

I think the result has `data`, not `bytes`.

```dart
final img = imglib.Image.fromBytes(width, height, result.data!);
```

**Reply by khoren93** [reply]
Thank you for bringing this to my attention. The code sample in the README has already been updated.
